arches_lingo/const.py

```diff
@@ -3,7 +3,26 @@
 
 TOP_CONCEPT_OF_NODE_AND_NODEGROUP = "bf73e5b9-4888-11ee-8a8d-11afefc4bff7"
 
-BROADER_NODE_AND_NODEGROUP = "f3f7bbea-0eb9-11ef-93db-0a58a9feac02"  # classification_status_ascribed_classification
+# classification_status
+BROADER_NODE_AND_NODEGROUP = "f3f7bbea-0eb9-11ef-93db-0a58a9feac02"
```

Comment:
@jacobtylerwalls heads up that Broader is no longer a single node nodegroup. This nodeid is only for the nodegroup, the primary data carrying node is on line 9 (`CLASSIFICATION_STATUS_ASCRIBED_CLASSIFICATION_NODEID`)